pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Customizing Kube-proxy on an OVHcloud Managed Kubernetes cluster
+slug: customizing-kubeproxy
+excerpt: 'Find out how to customize the Kube-proxy configuration on an OVHcloud Managed Kubernetes cluster'
+section: Network
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24; 
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   font-family: monospace !important;
+   font-size: 0.75em;
+   color: #ccc;
+ }
+ .small {
+     font-size: 0.75em;
+ }
+</style>
+
+**Last updated 1st February 2023**
+
+## Objective
+
+The OVHcloud Managed Kubernetes service provides you with Kubernetes clusters without the hassle of installing or operating them.
+
+The kube-proxy Kubernete's component (which runs on each Nodes and allows network comunication to Pods) with iptables is actually a bottleneck to scale the cluster to a high number of Nodes so at OVHcloud we decided to reduce this bottleneck and allow you to use kube-proxy with IPVS.
+
+[IPVS (IP Virtual Server)](https://kubernetes.io/blog/2018/07/09/ipvs-based-in-cluster-load-balancing-deep-dive/) is built on top of the Netfilter and implements transport-layer Load Balancing as part of the Linux kernel.
+
+At OVHcloud, we listen to our users and improve our products and services accordingly, which is why we give you the ability to customize the kube-proxy configuration.
+
+## Requirements 
+
+- An OVHcloud Managed Kubernetes cluster
+
+## Instructions
+
+### Configure kube-proxy through the API
+
+#### The API Explorer
+
+To simplify things, we are using the [API Explorer](https://api.ovh.com/) which allows to explore, learn and interact with the API in an interactive way.
+
+Log in to the API Explorer using your OVHcloud NIC handle.
+
+![Log in to the API Explorer](images/kubernetes-quickstart-api-ovh-com-001.png){.thumbnail}
+
+If you go to the [Kubernetes section](https://api.ovh.com/console/#/kube) of the API Explorer, you will see the available endpoints:
```

Comment:
Are you sure about this link ? It doesn't focus on any part of api.ovh.com

Reply:
thanks, it's an old endpoint that I need to fix in other tutos :)